app/src/main/res/values-land/dimens.xml

```diff
@@ -7,7 +7,7 @@
     <dimen name="video_item_search_title_text_size">16sp</dimen>
     <dimen name="video_item_search_duration_text_size">12sp</dimen>
     <dimen name="video_item_search_uploader_text_size">14sp</dimen>
-    <dimen name="video_item_search_upload_date_text_size">14sp</dimen>
```

Comment:
All of these values above are related to each other. If you change one, you should change others too, otherwise the text sizes will look off (as you can see in your screenshots, the uploader is bigger than the upload date).

How does it look like if you just remove all of these, so that [the corresponding smaller values in `values/dimens.xml`](https://github.com/TeamNewPipe/NewPipe/blob/bbf134bc6b19d0c1836029bce1181325dd54284b/app/src/main/res/values/dimens.xml#L17-L21) are used?

Please try out some sets of values (including the suggestion above) and provide some screenshots, so that we can choose the best.

Also, are you sure somebody using a TV or a tablet is able to read text so small?